xml/bp_chap_getting_started_with_salt_formulas.xml

```diff
@@ -450,7 +454,9 @@ timezone:
     <listitem>
      <para>
       <literal>$default</literal> allows you to specify a default value
-      that is displayed and used, if no other value is entered.
+      that is displayed and used, if no other value is entered.  In an
+      <literal>edit-group</literal> it allows to create members of the
```

Comment:
Not sure if I got this correctly:
```
In an <literal>edit-group</literal> it allows to create members of the group and populate them with specified data.
```

The `$default` in the `edit-group` is about default state of the members, I'd think it'd be nice to mention it here in some way, for instance:
```
In an <literal>edit-group</literal> it allows to create inital members of the group and populate them with specified data.
```
or

```
In an <literal>edit-group</literal> it allows to pre-populate members of the group and fill them with specified data.
```

Just my $.02